VoltAir/Engine/android/AndroidActivity.cpp

```diff
@@ -601,8 +602,14 @@ QKeyEvent* AndroidActivity::convertAndroidKeyEvent(int action, KeyCode keyCode)
     return new QKeyEvent(translatedAction, translatedKeyCode, Qt::NoModifier);
 }
 
-void AndroidActivity::onStudentUpdateReceiver(JNIEnv* env, jstring availableLetters, jstring availableNumbers)
+void AndroidActivity::onStudentUpdateReceiver(JNIEnv* env, jobject obj, jstring availableLetters, jstring availableNumbers)
 {
+    const char *userName = env->GetStringUTFChars(availableLetters,0);
+    QString letters(env->GetStringUTFChars(availableLetters, 0));
```

Comment:
@diegowald Could this be represented as an _array_ of strings instead of one single string?